pxr/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ if (EXISTS "${PROJECT_SOURCE_DIR}/pxr/exec")
     add_subdirectory(exec)
 endif()
 
-if ($(PXR_BUILD_USD_VALIDATION))
+if (${PXR_BUILD_USD_VALIDATION})
```

Comment:
@tallytalwar I believe this is a typo in `dev`

Reply:
Yup @beersandrew , it was fixed with last dev push: https://github.com/PixarAnimationStudios/OpenUSD/commit/1f0b71a66ab76868d59739074d0a473b33ee0f3f

Can you update your dev branch?